baseplate/clients/thrift.py

```diff
@@ -71,19 +103,19 @@ class ThriftContextFactory(ContextFactory):
 
     """
 
-    PROM_PREFIX = "bp_thrift_pool"
-    PROM_LABELS = ["client_cls"]
+    POOL_PREFIX = "thrift_pool"
```

Comment:
I didn't see these in the spec so I added them [here](https://github.snooguts.net/reddit/baseplate.spec/pull/49). To match the other connection pool metrics, the prefix should be `thrift_client_pool` with the label `thrift_pool` (matches redis)

Reply:
Fixed!